front/gatsby/src/ContextGate.tsx

```diff
@@ -24,7 +23,7 @@ export function ContextGate({
       <AccountsContextProvider>
         <TxQueueContextProvider>
           <ApiContextProvider provider={new WsProvider(ARCHIVE_NODE_ENDPOINT)}>
-            <StakingContextProvider>{children}</StakingContextProvider>
+            <>{children}</>
```

Comment:
```suggestion
            {children}
```

Reply:
Unfortunately, i get some error like https://github.com/paritytech/substrate-light-ui/pull/743/files#diff-418f37370656dbd0c860458c05c4933cR17

Reply:
ah, i guess that makes sense....